dracut/30ignition/ignition-generator

```diff
@@ -39,6 +39,7 @@ if $(cmdline_bool 'ignition.firstboot' 0); then
     add_requires ignition-disks.service
     add_requires ignition-files.service
     add_requires ignition-ask-var-mount.service
+    add_requires ignition-setup.service
```

Comment:
Maybe it'd be better to have the `-disks` and `-files` units depend on `-setup`, and drop the explicit install here?

Reply:
ok pushed a fixup that should address this.